apps/test/CommandHistoryTest.js

```diff
@@ -0,0 +1,78 @@
+var testUtils = require('./util/testUtils');
+var assert = testUtils.assert;
+
+describe("CommandHistory", function () {
+  var CommandHistory = require('@cdo/apps/CommandHistory');
+  var history, inputText;
+
+  beforeEach(function () {
+    history = new CommandHistory();
+    inputText = '';
+  });
+
+  it('moving back through history recounts commands in reverse order', function () {
```

Comment:
this is totally optional, but part of the fun of this test framework is writing statements that kind of read like english. For example: `it('recounts commands in reverse order when moving back through history', ...`